http_test.go

```diff
@@ -488,6 +488,17 @@ func TestHandlers(t *testing.T) {
 	req.Header.Set("Content-Type", "application/json")
 	h.ServeHTTP(recorder, req)
 	assert.Equal(t, http.StatusBadRequest, recorder.Code)
+	res = recorder.Result()
```

Comment:
Can we easily add checks to assert that no content body is being returned for the success cases?